src/scripts/main.js

```diff
@@ -1,8 +1,19 @@
 'use strict';
 
 const people = require('./lib/people');
+const dashboard = document.querySelector('.dashboard');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+[...people].forEach(({ name: fullName, sex, born, died }) => {
+  const tabelRow = document.createElement('tr');
 
-// write your code here
+  tabelRow.innerHTML = `
+    <td>${fullName}</td>
+    <td>${sex === 'm' ? 'Male' : 'Female'}</td>
+    <td>${born}</td>
+    <td>${died}</td>
+    <td>${died - born}</td>
+    <td>${Math.ceil(born / 100)}</td>
```

Comment:
```suggestion
    <td>${Math.ceil(died / 100)}</td>
```
![Screenshot from 2023-06-10 15-02-11](https://github.com/mate-academy/js_task_generate_table_DOM/assets/33029864/860aed75-2568-4231-bac4-43b0762776f1)